chart/values.schema.json

```diff
@@ -1066,6 +1076,44 @@
                     }
                 }
             }
+        },
+        "kerberos": {
+            "description": "Kerberos configurations for airflow",
+            "type": "object",
+            "properties": {
+                "enabled": {
+                    "description": "Enable kerberos.",
+                    "type": "boolean"
+                },
+                "ccacheMountPath": {
```

Comment:
```suggestion
                "cacheMountPath": {
```

Reply:
Same as the other comment -  The "ccache" is deliberate here.